src/js/_enqueues/admin/custom-background.js

```diff
@@ -83,7 +83,7 @@
 		 *
 		 * @return {void}
 		 */
-		$('#choose-from-library-link').click( function( event ) {
+		$('#choose-from-library-link').on( 'change', function( event ) {
```

Comment:
@Clorith should this be `.on( 'click', `
```suggestion
		$('#choose-from-library-link').on( 'click', function( event ) {
```